stylesheets/style.css

```diff
@@ -0,0 +1,2328 @@
+:root {
+  font-size: 1rem;
+}
+
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  margin: auto;
+  background-color: white;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "ARS Maquette Pro", Helvetica, Arial, sans-serif;
+  background-color: #fefefe;
+  padding-bottom: 10px;
+  overflow-x: hidden;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+select {
+  background-color: black;
+  color: grey;
+  border: none;
+}
+
+img {
+  height: 300px;
+  width: 100%;
+  object-fit: cover;
+}
+
+/* Small (-sm-) devices (landscape phones, 576px and up) */
+@media screen and (min-width: 400px) and (max-width: 768px) {
```

Comment:
You should only have two breakpoints, 768px and above for medium devices and 1024px and above for large devices.